include/cuco/detail/__config

```diff
@@ -32,3 +46,7 @@
 #if (CUCO_CUDA_MINIMUM_ARCH >= 700)
 #define CUCO_HAS_INDEPENDENT_THREADS
 #endif
+
+#if defined(__SIZEOF_INT128__) && (CUDART_VERSION) && (CUDART_VERSION >= 11060)
```

Comment:
Should I check for RT or CC version here?

Reply:
Educate me, please. IIUC, `int128` is supported since cuda 11.5 thus not sure why `CUDART` version is relevant. Is `__SIZEOF_INT128__` defined in `nv/target`?

Reply:
int128 support came with CTK 11.6 (preview in 11.5 through some compiler flag). I'm not sure if "introduced with 11.5" here means in CUDA runtime v11.5 or NVCC v11.5.

The __SIZEOF_INT128__ macro checks if the host compiler supports int128. If the CUDA version supports int128, but the host compiler doesn't, compilation will fail.